packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -101,9 +101,8 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
             {...buttonProps}
             className={css(classNames.slideBox, buttonProps!.className)}
             id={this._id}
-            disabled={disabled}
-            type="button"
             role="slider"
+            tabIndex={disabled ? -1 : 1}
```

Comment:
this one... not good. We should never set `tabIndex` to 1. Maybe 0, and with `data-is-focusable="true"`.

Reply:
Changed tabIndex to 0. Adding the react-a11y-tabindex-no-positive rule breaks if tabIndex is explicitly set to 1, but it doesn't break if tabIndex is conditionally set based on the disabled property.